httpev.ml

```diff
@@ -1119,6 +1119,21 @@ let setup_lwt config answer =
 let server_lwt config answer =
   Lwt_main.run @@ setup_lwt config answer
 
+let http_bind addr http_config =
```

Comment:
I think this function should be named `setup_<something>` because it's what it does. E.g. `setup_bind_lwt`.

Reply:
+ it can accept and apply the `answer` callback right away.